src/NexusMods.App.UI/Pages/MyLoadouts/GameLoadoutsSectionEntry/GameLoadoutsSectionEntryViewModel.cs

```diff
@@ -1,15 +1,130 @@
 using System.Collections.ObjectModel;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Avalonia.Threading;
+using DynamicData;
+using DynamicData.Binding;
 using NexusMods.Abstractions.GameLocators;
+using NexusMods.Abstractions.Games;
+using NexusMods.Abstractions.Loadouts;
 using NexusMods.App.UI.Controls.LoadoutCard;
+using NexusMods.App.UI.Pages.LoadoutGrid;
+using NexusMods.App.UI.Resources;
+using NexusMods.App.UI.Windows;
+using NexusMods.App.UI.WorkspaceSystem;
+using NexusMods.MnemonicDB.Abstractions;
+using ReactiveUI;
 
 namespace NexusMods.App.UI.Pages.MyLoadouts.GameLoadoutsSectionEntry;
 
 public class GameLoadoutsSectionEntryViewModel : AViewModel<IGameLoadoutsSectionEntryViewModel>, IGameLoadoutsSectionEntryViewModel
 {
-    public GameLoadoutsSectionEntryViewModel(GameInstallation gameInstallation)
+    private readonly CompositeDisposable _compositeDisposable;
+    private readonly GameInstallation _gameInstallation;
+    private readonly IWindowManager _windowManager;
+    private readonly ReadOnlyObservableCollection<IViewModelInterface> _loadoutCardViewModels;
+    private readonly ReadOnlyObservableCollection<IViewModelInterface> _cardViewModels;
+    private readonly SourceList<IViewModelInterface> _cardViewModelsSourceList = new();
+    public string HeadingText { get; }
+    public ReadOnlyObservableCollection<IViewModelInterface> CardViewModels => _cardViewModels;
+
+
+    public GameLoadoutsSectionEntryViewModel(GameInstallation gameInstallation, IConnection conn, IServiceProvider serviceProvider, IWindowManager windowManager)
     {
-        HeadingText = gameInstallation.Game.Name + " Loadouts";
+        _compositeDisposable = new CompositeDisposable();
+        _gameInstallation = gameInstallation;
+        _windowManager = windowManager;
+        HeadingText = string.Format(Language.MyLoadoutsGameSectionHeading, _gameInstallation.Game.Name);
+
+        Loadout.ObserveAll(conn)
+            .Filter(l => l.IsVisible() && l.InstallationInstance.LocationsRegister[LocationId.Game] == _gameInstallation.LocationsRegister[LocationId.Game])
+            .Transform(loadout =>
+                {
+                    return (IViewModelInterface)new LoadoutCardViewModel(loadout, conn, serviceProvider)
+                    {
+                        VisitLoadoutCommand = ReactiveCommand.Create(() => NavigateToLoadout(loadout)),
+                        CloneLoadoutCommand = ReactiveCommand.Create(() =>
+                            {
+                                // TODO: Implement Loadout cloning
+                            }
+                        ),
+                    };
```

Comment:
Don't use this pattern. Have the `LoadoutCardViewModel` implement these commands:

```csharp
ReactiveCommand.Create(() => Loadout);
```

Then set up a subscription to these commands to invoke `NavigateToLoadout`.

Check the docs for details.